ee/clickhouse/queries/funnels/funnel_correlation_persons.py

```diff
@@ -19,29 +19,44 @@
 
 
 class FunnelCorrelationActors:
-    def __init__(self, filter: Filter, team: Team, base_uri: str = "/") -> None:
+    def __init__(self, filter: Filter, team: Team, base_uri: str = "/", **kwargs) -> None:
         self._base_uri = base_uri
         self._filter = filter
         self._team = team
+        self._limit_actors = kwargs.get("limit_actors", True)
```

Comment:
Note: I would keep the named argument here since we're not forwarding it anywhere.